addons/csw/CfgWeapons.hpp

```diff
@@ -148,6 +145,9 @@ class CfgWeapons {
         model = QPATHTOEF(apl,ACE_CSW_Bag.p3d);
         modes[] = {};
         picture = QPATHTOF(UI\StaticAT_Icon.paa);
+        magazines[] = {
+            "Titan_AT"
```

Comment:
This does make them be able to load the magazines, doesn't it?

Reply:
> 
> 
> > I'll point them to the generic icons in csw if thats ok ?
> 
> No, use the RHS paths. If they changed them, adjust the paths. Obviously you have to start RHS to start these components.

I did, of course. With the RHS paths, the icon/picture shows up in the arsenal ok, but when you exit and look in your inventory, the error pops up and the launcher slot is blank.

Reply:
Yes, if you have it (the weapon bag) equipped, you can drag a mag to it, but nothing happens.

Reply:
That's terrible.

Reply:
For a Titan/Spike AT, i guess its pretty obvious what kind of mag you need for it, but adding compatible mags to the array is the only way to know what ammo you need for the weapon. Otherwise you have to scroll through the "All Magazines" list and pray you pick up the right one.

We can remove them, but the convenience of having the compatible mags listed under Magazines maybe outweighs the fact you can "placebo load" the weapon with them ?

Reply:
Maybe consider adding custom properties for magwells and magazine classes. Content of these properties could be read by ACE Arsenal and shown on the magazine list.